src/main/resources/application.yml

```diff
@@ -78,6 +78,7 @@ nakadi:
   auth:
     plugin:
       factory: org.zalando.nakadi.plugin.auth.DefaultApplicationServiceFactory
+  event.max.bytes: 1000000
```

Comment:
because you increased kafka broker message size to 3MB and expect that it is gonna be 3MB, but actually it only 1MB